src/optim_problems/run_opt.jl

```diff
@@ -59,23 +58,22 @@ what you can add to the opt_config:
 function run_opt(ts_data::ClustData,
                     opt_data::OptDataCEP,
                     opt_config::Dict{String,Any},
-                    fixed_design_variables::Dict{String,OptVariable};
-                    solver::Any=CbcSolver(),
+                    fixed_design_variables::Dict{String,Any},
```

Comment:
JLD2 doesn't allow a Dict{String,OptVariable} anymore after OptVariable improvement. Seems to be a bug in JLD2 - Hope that this can be reverted in the future to OptVariable again

Reply:
Is there an issue in JLD2 concerning this, or anybody else having this problem? Is this a general problem with structs within a Dict?

Reply:
It's a specific issue in JLD2 with Dicts containing elements with Vararg{}